locales/en/moves.json

```diff
@@ -101,7 +98,7 @@
     "assessment_answer": {
       "label": "{{question}} details"
     },
-    "documents": {
+    "z": {
```

Comment:
Is this change intentional?

Reply:
Nope.  I must have hit a key or something ...